frontend/src/metabase/components/Select.jsx

```diff
@@ -4,11 +4,22 @@ import ColumnarSelector from "metabase/components/ColumnarSelector.jsx";
 import Icon from "metabase/components/Icon.jsx";
 import PopoverWithTrigger from "metabase/components/PopoverWithTrigger.jsx";
 
+import cx from "classnames";
+
 export default class Select extends Component {
+    constructor(props, context) {
+        super(props, context);
+        this.state = {
+            value: null
+        };
+    }
+
     static propTypes = {
         value: PropTypes.any,
+        changeOnSelect: PropTypes.bool,
         options: PropTypes.array.isRequired,
         placeholder: PropTypes.string,
+        triggerClasses: PropTypes.string,
```

Comment:
True. Actually it looks like we're applying className to both the popover and trigger, which seems wrong. https://github.com/metabase/metabase/blob/e8cb3f9f3082a801f53d08fdd1376461bb8b3275/frontend/src/metabase/components/Select.jsx#L80-L82

Maybe just make a note saying we should go back and clean this up.